graylog2-web-interface/src/views/components/visualizations/utils/formatValueWithUnitLabel.ts

```diff
@@ -15,9 +15,9 @@
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
 
-import { DECIMAL_PLACES } from 'views/components/visualizations/Constants';
 import getUnitTextLabel from 'views/components/visualizations/utils/getUnitTextLabel';
+import { formatNumber } from 'util/NumberFormatting';
 
-const formatValueWithUnitLabel = (value: number | string, abbrev: string) => `${Number(value).toFixed(DECIMAL_PLACES)} ${getUnitTextLabel(abbrev)}`;
+const formatValueWithUnitLabel = (value: number | string, abbrev: string) => `${formatNumber(Number(value), { minimumDigits: 1 })} ${getUnitTextLabel(abbrev)}`;
```

Comment:
It was a product requirement to have only 1 digit here. We can use `{ digits: DECIMAL_PLACES } ` here, but then we will have 0 whenever the value is small. In this specific case, it will most likely not happen, because it will be more than 1 in most of the cases. But lets cover this case as well. We can provide `maximumDigits` in `formatNumber` options to have {minimumDigits: DECIMAL_PLACES, maximumDigits: DECIMAL_PLACES} or add the same logic to `digits` and use in original format function digits or minimal digits to express the value

Reply:
If we want to have only 1 digit, we should format it like that regardless if there is a unit configured or not. We can adjust this in other places to and make it a general convention to have consistent formatting of numbers everywhere, regardless if it's 1 or 2 digits by default.

Regarding very small values, maybe it was not considered that we would end up with `0.0` or `0.1` for everything smaller than `0.05` or `0.1`, so we should discuss this with @Emy-01? The PR is not pressing, so we can wait.

Reply:
Talked to @Emy-01 and he is also in favor of making it consistent, adding the special case for very low numbers.